contracts/safe.clar

```diff
@@ -242,6 +242,7 @@
                 (confirmations (get confirmations tx))
             )
 
+            (asserts! (is-eq (get confirmed tx) false) ERR-TX-CONFIRMED)
```

Comment:
I think boolean operator `not` fits here a bit better and it is a bit cheaper, but that's just my personal preference.

```clarity
(asserts! (not (get confirmed tx)) ERR-TX-CONFIRMED)
```